src/test/java/org/mifos/integrationtest/cucumber/stepdef/MockFlowTestDef.java

```diff
@@ -23,25 +23,29 @@ public class MockFlowTestDef extends BaseStepDef {
     @Autowired
     OperationsAppConfig operationsAppConfig;
 
+    @Autowired
+    ScenarioScopeDef scenarioScopeDef;
```

Comment:
rename variable to scenarioScopeState or scenarioScopeCintext to match its purpose